pom.xml

```diff
@@ -1314,6 +1313,7 @@
     <bouncycastle.version>1.46</bouncycastle.version>
     <kerby.version>1.0.1</kerby.version>
     <hbase.shaded.gson.version>3.0.0</hbase.shaded.gson.version>
+    <hbase.shaded.miscellaneous.version>3.0.0</hbase.shaded.miscellaneous.version>
```

Comment:
The latest is 3.4.0. 
I just copied the version supplied for hbase-shaded-gson here. 

Both probably should be updated. 

Based on discussion elsewhere on the PR, currently all released versions of hbase-shaded-miscellaneous are unsuitable for branch-1 because the shaded Guava's class files require Java 8 or above to execute. Presumably a new release of hbase-shaded artifacts suitable for Java 7 can be done and this will change to reflect that version. (And there might be an artifact name change, or some trick with classifiers.)